app/components/workbench/Workbench.client.tsx

```diff
@@ -313,7 +314,11 @@ export const Workbench = memo(({ chatStarted, isStreaming, actionRunner }: Works
 
     await shell.ready();
 
-    await shell.executeCommand(Date.now().toString(), 'pnpm install && npx -y @agent8/deploy && pnpm run dev');
+    if (localStorage.getItem(SETTINGS_KEYS.AGENT8_DEPLOY) === 'false') {
```

Comment:
is `false` intended?

Reply:
yes it's because default value is true, In other words, it is true even if there is no value.

Reply:
Well.. then, is `npx -y @agent8/deploy` in below line also intended?